components/metadata-service/cmd/metadataservice/metadataservice.go

```diff
@@ -56,12 +56,12 @@ func main() {
 	)
 
 	if err != nil {
-		log.Errorf("Unable to initialize: '%s'", err.Error())
+		log.Errorf("Metadata Service: unable to initialize, %s", err.Error())
```

Comment:
There is an inconsistency in the naming scheme used. There are two conventions used in the code:
- prefix designates go file containing the implementation (e.g. "Metadata Service")
- prefix designates the operation within go file (e.g. "monitoring:")

Reply:
Ok, I'll take care of it.